modules/DirectControl/src/directcontrol.cpp

```diff
@@ -215,7 +214,8 @@ size_t DirectControl::handleRDCAMessage(
 	ssize_t bytesRead = decodeRDCAMessage(byteData.data(), &recvMessage, byteData.size(), currentTime, false);
 	if (bytesRead >= 0) {
 		byteData.erase(byteData.begin(), byteData.begin() + bytesRead);
-		DataDictionarySetRequestedControlAction(recvMessage.executingID, &recvMessage);
+		//TODO: Implement equivalent of below line in using ROS2.
```

Comment:
Is this something that needs to be done before we merge?

Reply:
The code is not relevant anymore, can be implemented using equivalent ros2 features. Line was removed since datadictionary was deleted in this PR